packages/rocketchat-i18n/i18n/assistifyAI.de.i18n.json

```diff
@@ -11,6 +11,8 @@
     "Assistify_AI_Smarti_Domain_Description": "Trage hier den Smarti Client-Name ein, der zu dieser Rocket.Chat Instanz gehört.",
     "Assistify_AI_Smarti_Auth_Token": "Smarti Client-Token",
     "Assistify_AI_Smarti_Auth_Token_Description": "Trage hier den Smarti Client-Token ein, damit sich Rocket.Chat gegenüber Smarti authentifizieren kann.",
+    "Assistify_AI_Smarti_Inline_Highlighting_Enabled": "Hervorheben identifizierter Termini",
```

Comment:
We should use `Terme` here like in the message below.  [Duden](https://www.duden.de/rechtschreibung/Term)